mapzen-places-api/src/main/java/com/mapzen/places/api/Place.java

```diff
@@ -1,14 +1,23 @@
 package com.mapzen.places.api;
 
 import android.net.Uri;
+import android.os.Parcelable;
 
 import java.util.List;
 import java.util.Locale;
 
 /**
  * Represents a physical place selected from a {@link com.mapzen.android.graphics.MapzenMap}.
  */
-public interface Place {
+public interface Place extends Parcelable {
+
+  public static final int TYPE_LOCALITY = 1009;
+  public static final int TYPE_ADMINISTRATIVE_AREA_LEVEL_3 = 1003;
+  public static final int TYPE_SUBLOCALITY = 1022;
+  public static final int TYPE_POSTAL_CODE = 1015;
+  public static final int TYPE_COUNTRY = 1005;
+  public static final int TYPE_ADMINISTRATIVE_AREA_LEVEL_1 = 1001;
+  public static final int TYPE_ADMINISTRATIVE_AREA_LEVEL_2 = 1002;
```

Comment:
Are we able to effectively determine place type at this time when fetching details from Pelias?

Or should we omit these constant values in v1 so as to not mislead developers into thinking this data will be valid in the response?